### PR TITLE
Upgrade to Pandora 6

### PR DESCRIPTION
**Pull request by wgmouton**

### CODE REVIEW

File: impacthub-4/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# ixo Network Validator for Impacthub-3
```
Comment (alwyn-ixo) [inline impacthub-4/README.md:1]: @wgmouton This still mentions Impacthub-2; is that correct?

File: impacthub-4/InstallImpacthub.sh [hunk 2]
```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
```
Comment (alwyn-ixo) [inline impacthub-4/InstallImpacthub.sh:1]: @wgmouton Should this not be ixo-4?

File: impacthub-4/README_INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
+# ixo Network Validator: Impacthub-3 Setup
```
Comment (alwyn-ixo) [inline impacthub-4/README_INSTALL.md:1]: @wgmouton Should this be impacthub-3 or ixo-4?

File: impacthub-4/README_UPGRADE.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
+# Upgrade Instructions: ImpactHub-2 -> ImpactHub-3
```
Comment (alwyn-ixo) [inline impacthub-4/README_UPGRADE.md:1]: @wgmouton impacthub-2 to -3 or ixo-4?

**Review by YBAAC — APPROVED** [review]

**Review by YBAAC — APPROVED** [review]

**Pull request merged**